book/website/collaboration/inclusive-events/inclusive-events-socials.md

```diff
@@ -0,0 +1,146 @@
+(cl-inclusive-events-social-and-networking-guide)=
+# Inclusive Social and Networking Events
+Although your event's main content and sessions will be well thought out, interesting, and engaging for attendees, many attendees often speak highly of the social and collaborative environments that networking events bring to large events. 
```

Comment:
I would delete this sentence as it doesn't contain that much info and start with the second sentence which provides a stronger argument.